packages/components/pie-cookie-banner/README.md

```diff
@@ -86,7 +86,7 @@ yarn dev --filter=pie-storybook
 
 | Property | Type | Default | Description |
 |---|---|---|---|
-| - | - | - | - |
+| hasPrimaryActionsOnly | `Boolean` | `false` | When true, sets the "Necessary only" button variant to "primary". |
```

Comment:
I'd probably avoid using copy in the description here in case it changes. Could be:

`When true, sets all action buttons in the cookie banner to display as variant "primary".`

Reply:
Yeh, I think we'll definitely need to add some docs somewhere about this when we start integrating.

Totally get that some events etc are fired as necessary only right now, just figured as this one could be uncoupled may be bette me to do that potentially.